core/templates/services/svg-sanitizer.service.spec.ts

```diff
@@ -400,4 +490,38 @@ describe('SvgSanitizerService', () => {
         testCase.expected[1], 'Tags - Case:' + testCase.title);
     });
   });
+
+  it('should return correct GitHub issue URL', () => {
```

Comment:
please add when it should do so in the `it` description.

Reply:
Done!